packages/hdwallet-portis/src/portis.ts

```diff
@@ -223,18 +223,16 @@ import {
       return Promise.resolve()
     }
 
+    // TODO this needs to handle more than just eth
     public async getPublicKeys(msg: GetPublicKey[]): Promise<PublicKey[]> {
       console.log('GET PUBLIC KEYS')
-
-      const portisResult = await this.portis.getExtendedPublicKey("m/44'/60'/0'");
+      const portisResult = await this.portis.getExtendedPublicKey("m/44'/60'/0'")
```

Comment:
We should probably ask Portis folks for an API that gives more than one at a time.

Reply:
That way if they want to add some permission dialog, we don't have to bug the user over and over.